test/token/ERC721/extensions/ERC721Votes.test.js

```diff
@@ -382,13 +382,12 @@ contract('ERC721Votes', function (accounts) {
     describe('numCheckpoints', function () {
       it('returns the number of checkpoints for a delegate', async function () {
 
-        await this.token.transferFrom(holder, recipient, NFT1, { from: holder }); //give an account two tokens for readability
+        await this.token.transferFrom(holder, recipient, NFT1, { from: holder }); //give an account two tokens for readability        
```

Comment:
There are some trailing whitespaces on line 385 and 388. Does that pass the linter ? IMO it shouldn't

Reply:
Solved with commit 51002f9